buildstockbatch/base.py

```diff
@@ -583,6 +585,33 @@ def validate_reference_scenario(project_file):
 
         return True  # Only print the warning, but always pass the validation
 
+    @staticmethod
+    def validate_resstock_version(project_file):
+        """
+        Checks the minimum required version of BuildStockBatch against the version being used
+        """
+        cfg = get_project_configuration(project_file)
+
+        buildstock_rb = os.path.join(cfg['buildstock_directory'], 'resources/buildstock.rb')
+        if os.path.exists(buildstock_rb):
+            versions = {}
+            with open(buildstock_rb, 'r') as f:
```

Comment:
I kinda feel uneasy about string searching the buildstoc.rb file to find out the ResStock_Version and BuildStockBatch_Version, but don't really know a better way!